internal/testutil/pkgdata.go

```diff
@@ -1,152 +1,536 @@
 package testutil
 
 import (
-	"encoding/base64"
+	"archive/tar"
+	"bytes"
+	"embed"
+	"path/filepath"
+	"time"
+
+	"github.com/blakesmith/ar"
+	"github.com/klauspost/compress/zstd"
 )
 
 var PackageData = map[string][]byte{}
 
 func init() {
-	baseFilesData, err := base64.StdEncoding.DecodeString(baseFilesBase64)
+	for name, entries := range pkgs {
+		for i, _ := range entries {
+			if !entries[i].noFixup {
+				fixupEntry(&entries[i])
+			}
+		}
+		data, err := buildDeb(entries)
+		if err != nil {
+			panic(err)
+		}
+		PackageData[name] = data
+	}
+}
+
+type tarEntry struct {
+	header  tar.Header
+	noFixup bool
+	content []byte
+}
+
+var pkgs = map[string][]tarEntry{
+	"base-files": {
+		{
+			header: tar.Header{
+				Name: "./",
+			},
+		},
+		{
+			header: tar.Header{
+				Name: "./bin/",
+			},
+		},
+		{
```

Comment:
The standard convention for formatting such lists looks like this:
```
"base-files": {{
        header: tar.Header{
                Name: "./bin/",
        }, {
                Name: "...",
        }, {
                ...
        }},
}}
```
This saves lots of empty spacing that makes it harder to go through the list.

Reply:
as a side note, I think [this PR](https://github.com/canonical/chisel/pull/31/files) could help keep the consistency for formatting